tests/components/destination_list.spec.js

```diff
@@ -87,7 +87,22 @@ describe("DestinationList", () => {
     expect(tree).toMatchSnapshot();
   });
 
-  test("passed selectedItems", () => {
+  test("passed selectedItems with grouping", () => {
```

Comment:
i think it will be good to have a snapshot tests for case withGrouping on and empty list of items as well